src/cart/SConscript

```diff
@@ -56,6 +68,7 @@ def macro_expand(target, source, env):
 def scons():
     """Scons function"""
 
+    print(" SAB: in src/cart: build library")
```

Comment:
This looks like a debug statement left in

Reply:
e94b325b546c7fef70ddfacf9a6428130193644d